modules/azure/api_management_api/variables.tf

```diff
@@ -172,6 +172,12 @@ variable "soap_pass_through" {
   default     = false
 }
 
+variable "api_type" {
+  type        = string
+  description = "Type of API. Possible values are graphql, http, soap, and websocket. Defaults to http"
+  default     = "http"
```

Comment:
based on my suggestion to not break other peoples code, I would put to null as default